backend/packages/identity-scan/src/scan/jobs/index.js

```diff
@@ -0,0 +1,58 @@
+const {
+  identity: { getRegistrarsCollection },
+} = require("@statescan/mongo");
+const {
+  chain: { getApi },
+} = require("@osn/scan-common");
+const { getIdentityStorage } = require("../utils/getIdentityStorage");
+
+async function bulkUpdateRegistrars(registrars) {
+  const identityCollection = await getRegistrarsCollection();
+
+  const operations = registrars.map((identity) => ({
+    updateOne: {
+      filter: { _id: identity.registrarIndex },
+      update: { $set: identity },
+      upsert: true,
+    },
+  }));
+
+  await identityCollection.bulkWrite(operations);
+}
+
+async function initRegistrars() {
+  const api = await getApi();
+  await getAllRegistrars(api);
+}
+
+async function getAllRegistrars(api) {
```

Comment:
Bad naming. We are not getting registrars and return them from this function, but saving them to database. So I think `saveAllRegistrars`.